commands/auth.go

```diff
@@ -161,7 +160,7 @@ func loginCmdF(cmd *cobra.Command, args []string) error {
 	if username != "" && password == "" {
 		stdinPassword, err := getPasswordFromStdin()
 		if err != nil {
-			return errors.New("Couldn't read password. Error: " + err.Error())
+			return fmt.Errorf("couldn't read password: %v", err)
```

Comment:
Go 1.13 added support for error wrapping. To me this is the new default way to handle multiple layers of error. This can either be done using https://github.com/pkg/errors or the `%w` verb. In this example it would be:
```suggestion
			return errors.WithMessage(err, "couldn't read password")
```
Or
```suggestion
			return fmt.Errorf("couldn't read password: %w", err)
```

I personally really dislike the `fmt` style, but I would be fine committing to it. Either way, I'm 3/5 that we should use one of these two ways.

Reply:
The idea was to stick on standard library. It's a little confusing when using both `errors` and `github.com/pkg/errors`. Also, I agree we can wrap errors with `%w` if we are going to unwrap them.

Reply:
> The idea was to stick on standard library.

Why? Don't get me wrong, the std lib is great, but I think it's lacking some feature `github.com/pkg/errors` has, like stack printing on errors.

> It's a little confusing when using both `errors` and `github.com/pkg/errors`

Well, we could only use `github.com/pkg/errors` :wink:

Reply:
> Don't get me wrong

No problem, the idea was to discuss these ideas ☺️

> Well, we could only use `github.com/pkg/errors` 😉

I agree we can use only one of them.

Reply:
@isacikgoz Would you be fine converging on `github.com/pkg/errors` and use `errors.WithMessage` and `errors.WithMessagef` respectively?

Reply:
Sure, I think it is fine. Also, using `github.com/pkg/errors` won't introduce any indirect dependencies which is a good thing.

Reply:
Then let's go this path :wink: